package.json

```diff
@@ -4,20 +4,33 @@
   "description": "Charting library that relies on React's virtual diffing.",
   "main": "lib/index.js",
   "module": "es/index.js",
-  "files": ["css", "es", "lib", "umd"],
+  "files": [
+    "css",
+    "es",
+    "lib",
+    "umd"
+  ],
   "scripts": {
     "clean": "nwb clean-module && nwb clean-demo",
     "start": "nwb serve-react-demo",
     "test": "nwb test-react",
     "build": "nwb build",
-    "build:watch":
-      "nodemon -w src -x 'rm .babelrc &> /dev/null; nwb build-react-component --no-demo'",
+    "build:watch": "nodemon -w src -x 'rm .babelrc &> /dev/null; nwb build-react-component --no-demo'",
     "test:coverage": "nwb test-react --coverage",
     "test:watch": "nwb test-react --server",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "storybook:build": "build-storybook -c .storybook -o .out",
     "demo:build": "nwb build-react-component src/index.js",
-    "package:deploy": "yarn build && np"
+    "package:deploy": "yarn build && np",
+    "lint": "eslint src",
+    "precommit": "lint-staged"
```

Comment:
I would strongly encourage us to use "prepush" instead of "precommit." Or maybe even "postcommit".

In short, I don't like things getting in the way of commits because that is both annoying and dangerous. If it prevents a commit, that can result in data loss.